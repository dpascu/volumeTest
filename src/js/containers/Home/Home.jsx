import React from 'react';

import Video from '../../components/Video';

const SRC_VIDEO = '../../../media/unmade.mp4';

const Home = () => (<Video src={SRC_VIDEO} />);

export default Home;
