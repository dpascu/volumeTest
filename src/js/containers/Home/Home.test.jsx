/* global describe it expect */
import React from 'react';
import renderer from 'react-test-renderer';

import Home from './Home';

describe('Container Home', () => {
  it('should render container', () => {
    const container = renderer.create(
      <Home />
    );

    const tree = container.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
