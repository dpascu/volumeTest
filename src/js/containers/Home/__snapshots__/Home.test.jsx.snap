// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Home should render container 1`] = `
Array [
  <div
    className="layer left-up"
  >
    <div
      className="weather"
    >
      <img
        alt="loading"
        src="../../../img/loader.gif"
      />
    </div>
  </div>,
  <video
    autoPlay={true}
    className="video-full"
  >
    <track
      kind="captions"
    />
    <source
      src="../../../media/video.mp4"
      type="video/mp4"
    />
    Your browser does not support the video tag.
  </video>,
]
`;
